app/models/user.rb

```diff
@@ -29,6 +29,14 @@ def to_s
     "#{first_name} #{last_name}"
   end
 
+  def admin
+    (context == AircraftPolicy::AIRCRAFT_PERM_STRING) ? "1" : "0"
+  end
+
+  def admin=(value)
+    self.context = (value == "1") ? AircraftPolicy::AIRCRAFT_PERM_STRING : ""
```

Comment:
```suggestion
    self.context = AircraftPolicy::WING_ADMIN_PERMISSION if value == "1"
```

Reply:
Will this change clear the value, when Admin is taken away?

Reply:
Ah yes, the case where the user's admin permission is revoked slipped my mind.
- in that case then
```suggestion
    self.context = (value == SET_ADMIN) ? AircraftPolicy::WING_ADMIN_PERMISSION : ""
```
where `SET_ADMIN = 1` is in `user.rb`.